JustLog/Classes/Logger.swift

```diff
@@ -50,6 +50,11 @@ public final class Logger: NSObject {
      */
     public var allowUntrustedServer: Bool = false
 
+    /**
+     Default to `false`, if `true` ssl is not required for socket 
```

Comment:
Comments here can be avoided, this one has the values inverted (but assuming the above changes are done, this would go too).